src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -92,11 +102,28 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/user-page.html?user=" + recipient);
   }
 
-  /** 
+  /**
    * Private method to recognize the image url in the text message and convert to <img> tag.
    * Message with no img url will not be modified.
    */
   private String extractImgUrl(String text) {
     return (text.replaceAll(this.imgUrlRegex, this.imgUrlReplacement));
   }
+
+  /**
+   * Translates messages based on a URL query parameter.
```

Comment:
This comment is misleading. You are not doing it based on the URL parameter, you are doing it based on the argument value.

Reply:
That's right, I'll fix that thank you!